src/perturbed/abstract_perturbed.jl

```diff
@@ -1,7 +1,8 @@
 """
-    AbstractPerturbed
+    AbstractPerturbed{B}
```

Comment:
I would give a more explicit name to `B`, like `parallel`

Reply:
Or at least `P` if you're too lazy

Reply:
I like parallel here